src/applications/financial-status-report/components/monetary/CashOnHand.jsx

```diff
@@ -4,6 +4,7 @@ import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButto
 import { VaNumberInput } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
 import { isValidCurrency } from '../../utils/validations';
 import { currency as currencyFormatter } from '../../utils/helpers';
+import { safeNumber } from '../../utils/calculateIncome';
 
```

Comment:
making a mental note to change this to utils/helpers before caclculateExpenses gets merged